frontendActions/src/main/resources/GitMacheteBundle.properties

```diff
@@ -53,6 +53,11 @@ action.GitMachete.BaseRebaseBranchOntoParentAction.text=_Rebase Branch onto Pare
 action.GitMachete.BaseResetBranchToRemoteAction.description.enabled={0} branch ''{1}'' to its remote tracking branch
 action.GitMachete.BaseResetBranchToRemoteAction.action-name=Re_set to Remote
 action.GitMachete.BaseResetBranchToRemoteAction.description-action-name=Reset to remote
+action.GitMachete.BaseResetBranchToRemoteAction.info-dialog.dont-show-again=Don't Show Again
+action.GitMachete.BaseResetBranchToRemoteAction.info-dialog.title=Git Machete Reset Info
+action.GitMachete.BaseResetBranchToRemoteAction.info-dialog.ok-text=Reset
+# TODO (#444): prohibit reset on conflicts with uncommitted
+action.GitMachete.BaseResetBranchToRemoteAction.info-dialog.message=<html>Git Machete Reset performs <b>git reset --keep</b>.<br>In case of uncommitted changes you will be asked to resolve File Cache Conflicts.<br>Locally committed changes diverged from the remote will be lost.</html>
```

Comment:
Hmm I don't really like the `...will be lost` phrasing :/

Reply:
Maybe sth along the lines `<branch-name> will point to the same commit as its remote branch. Use `git reflog <branch-name> to see the history of commits pointed by '<branch-name>'`... a bit awkard, I guess a better formulation is possible